recipe/meta.yaml

```diff
@@ -132,6 +141,43 @@ outputs:
       imports:
         - gym.envs.robotics
 
+  - name: gym-toy_text
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('gym', exact=True) }}
+        - scipy >=1.4.1
+    test:
+      imports:
+        - gym.envs.toy_text
+
+  - name: gym-other
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('gym', exact=True) }}
+        - lz4 >=3.1.0
+        - opencv-python >=3.0
+    test:
+      imports:
+        # There are no tests in the source code for gym-other
+
+  - name: gym-accept-rom-license
+    requirements:
+      host:
+        - python
+      run:
+        - python
+        - {{ pin_subpackage('gym', exact=True) }}
+        - autorom[accept-rom-license] ~=0.4.2
```

Comment:
I'm doubtful about the use of adding this subpackage - what does it do?

Also, I'm very surprised that `autorom[accept-rom-license]` works as a conda package - are we sure this works as intended?